product_docs/docs/net_connector/8.0.5.1/01_release_notes.mdx

```diff
@@ -8,7 +8,10 @@ The EDB .NET Connector provides connectivity between a .NET client application a
 
 New features, enhancements, bug fixes, and other changes in the EDB .NET Connector `8.0.5.1` include:
 
-| Type | Description |
-|------|-------------|
-|      |             |
-|      |             |
\ No newline at end of file
+| Type           | Description                                                                                                                                                                                                                                        | Addresses |
+|----------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-----------|
+| Upstream merge | Merged with community .NET driver version 8.0.5 and EF Core Driver 8.0.10. See [release notes](https://www.npgsql.org/doc/release-notes/8.0.html) for more information about merge updates.                                                        |           |
+| Bug fix        | Fixed a performance issue. Performance is now improved when reading data while targeting .NET Framework 4.7.2, 4.8, and 4.8.1.                                                                                                                     | #41979    |
+| Enhancement    | Added support for EDB Postgres Advanced Server 17.2.                                                                                                                                                                                               |           |
+| Enhancement    | Added support for `IS TABLE OF`. EDB Postgres Advanced Server supports Oracle nested table collection types created with `CREATE TYPE ... AS TABLE OF` statements. See [Using nested tables](./using_nested_table_types.mdx) for more information. |           |
+| Deprecation    | Removed .NET5, .NET6, and .NET7 targets as they have reached end of support.                                                                                                                                                                       |           |
```

Comment:
@xfischer  does this mean we should remove .NET 6.0 and .NET 7.0 from this page? 

https://deploy-preview-6219--edb-docs-staging.netlify.app/docs/net_connector/latest/03_the_advanced_server_net_connector_overview/ 

Or are those different things?

Reply:
Good catch! Yes they should be also removed